### PR TITLE
Add all necessary text and images 

### PR DESCRIPTION
**Pull request by amuamushu**
Add the text from the walkthrough and some of the additional text from the customization list as well as the image from the walkthrough.

https://screenshot.googleplex.com/ouSEeQNbuux

### CODE REVIEW

**Comment by amuamushu** [comment]
Add travel images to site and format them
https://screenshot.googleplex.com/xSa1iQhhOZC.png

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -11,6 +11,28 @@
       <h1>Amy's Portfolio</h1>
       <p>Hi! I'm Amy and welcome to my portfolio!</p>
       <p>I'm a rising junior at UC San Diego studying Data Science!</p>
+      <h2>Background</h2>
+      <p>I am a four-time instructional assistant for the data structures
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html:17]: Is "four" a typo?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -11,6 +11,28 @@
       <h1>Amy's Portfolio</h1>
       <p>Hi! I'm Amy and welcome to my portfolio!</p>
       <p>I'm a rising junior at UC San Diego studying Data Science!</p>
+      <h2>Background</h2>
+      <p>I am a four-time instructional assistant for the data structures 
+         and algorithms course at my university. I absolutely love it 
+         because I get to help people learn about my favorite subject 
+         and I find it extremely rewarding being able to help information 
+         click for someone.
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html:21]: I am not sure what "help information click for someone" means. Can you rephrase?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -44,6 +44,9 @@ <h2>List of Projects:</h2>
       <button onclick="addRandomGreeting()">Quote</button>
       <div id="greeting-container"></div>
     </div>
-    <img src="images/me.JPG" height="500px">
+    <h2>Connect with Me!</h2>
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html]: nit: "me"

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/index.html:17]: I don't think so. I meant four-time as in I tutored for four iterations of this course. I'll remove it anyways because I think it's unnecessary.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/index.html]: Yes, I reworded it to say "help students finally understand the material."

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (amuamushu) [reply portfolio/src/main/webapp/index.html]: Renamed the file to amyProfile.jpg

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -18,7 +18,7 @@ <h2>Background</h2>
          and algorithms course at my university. I absolutely love it 
          because I get to help people learn about my favorite subject 
          and I find it extremely rewarding helping students finally 
-         make sense for students. 
+         understand the material.
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html:21]: nit: to help students understand the course material.

Comment (amuamushu) [reply portfolio/src/main/webapp/index.html]: I reworded it. Thanks!
